packages/react-sdk/src/css/presets.ts

```diff
@@ -25,3 +25,102 @@ export const outline: Styles = [
     value: { type: "unit", value: 1, unit: "px" },
   },
 ];
+
+export const margins = [
+  {
+    property: "marginTop",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginRight",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginBottom",
+    value: { type: "unit", value: 0, unit: "px" },
+  },
+  {
+    property: "marginLeft",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+] satisfies Styles;
+
+export const blockquote = [
+  ...margins,
+  {
+    property: "paddingTop",
+    value: { type: "unit", value: 10, unit: "px" },
+  },
+  {
+    property: "paddingBottom",
+    value: { type: "unit", value: 10, unit: "px" },
+  },
+  {
+    property: "paddingLeft",
+    value: { type: "unit", value: 20, unit: "px" },
+  },
+  {
+    property: "paddingRight",
+    value: { type: "unit", value: 20, unit: "px" },
+  },
+  {
+    property: "borderLeftWidth",
+    value: { type: "unit", value: 5, unit: "px" },
+  },
+  {
+    property: "borderLeftStyle",
+    value: { type: "keyword", value: "solid" },
+  },
+  {
+    property: "borderLeftColor",
+    value: { type: "rgb", r: 226, g: 226, b: 226, alpha: 1 },
+  },
+] satisfies Styles;
+
+export const h1 = [
+  ...margins,
```

Comment:
headers have only top and bottom margins set. Do we want to bloat styles with left right too?